src/controllers/v2/note.js

```diff
@@ -0,0 +1,55 @@
+import database from '../../models/index.js';
+
+const {Note} = database;
+
+const NoteController = {
+  /**
+   * The create function writes the incoming Note to the appropriate database tables.
+   *
+   * @param {any } appId The application that the Note will be based on.
+   * @param {any | undefined} incomingNote The Note details.
+   * @returns {any} Returns newNote, the newly created Note.
+   */
+  // prettier-ignore
+  async create (appId, incomingNote) {
+    let newNote;
+    // Start a transaction.
+    await database.sequelize.transaction(async (t) => {
+      incomingNote.ApplicationId = appId;
+
+      // Add the Note to the DB.
+      newNote = await Note.create(incomingNote, {transaction: t});
+    });
+
+    // If all went well and we have a new application return it.
+    if (newNote) {
+      return newNote;
+    }
+
+    // If no new application was added to the DB return undefined.
+    return undefined;
+  },
+
+  /**
+   * Retrieve the notes from the database.
+   *
+   * @param {number} id An existing sett's ID.
```

Comment:
Setts ID?

Reply:
Done

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/77/commits/e33b7df9f6afa48d4e7d2ef0bd9782a3a13aa654